gammapy/maps/region/geom.py

```diff
@@ -513,6 +513,14 @@ def pix_to_idx(self, pix, clip=False):
         return tuple(idxs)
 
     def coord_to_pix(self, coords):
+        # inherited docstring
+
+        if isinstance(coords, dict):
+            coords.setdefault("skycoord", self.center_skydir)
```

Comment:
what happens here if the `dict` contains already `lat` and `lon` keys?

Reply:
Then `.setdefault()` does nothing...

Reply:
It will add the `skycoord` to the list of keys and it will later be listed as one of the axes.

Reply:
Yes, of course! I forgot we treat `skycoord` separately from `lon` and `lat`. It was such a bad idea to support all of those, like `MapCoord`, `dict` (with `skycoord` and `lon` and `lat`) and `tuple`...it's so bad to maintain. We should get rid of it for Gammapy v2.0...